pokemon-checker/src/services/PokemonService.ts

```diff
@@ -87,7 +86,14 @@ export class PokemonService {
    * Lookup an individual pokemon
    * @param url the url for the specific pokemon
    */
-  public getPokemon(url: string): Promise<any> {
-    return fetch(url).then((response) => response.json());
-  }
+  public getPokemon = async (url: string) => {
```

Comment:
All of the `async` and `await` keywords were added automatically by ESLint.

I still don't fully grasp how it works, but it does feel cleaner than writing `.then()` 3 times in a row.

Reply:
Mine doesn't recommend either. Generally `async/await` is cleaner
`async ` tells the function that its returning a `Promise`, (it will automatically wrap whatever is returned as a `Promise` if a `Promise` isn't returned).

`await` can only be added in a function that is delcared as `async` and it makes the code wait until the Promise resolves until it continues.

There isn't any performance increase with `async/await` vs .then chaining, however the former generally looks neater
https://blog.pusher.com/promises-async-await/